### ISSUE TITLE
Scratchpad: Autosaving & Window Focus

### ISSUE THREAD

**Issue by beauwest** [zeitlings/alfred-workflows#25]
If this is the wrong place to discuss this, feel free to close!

First, the Scratchpad is great. I've been using a simple app called FromScratch for years, but it's no longer maintained and this has the potential to replace it for me.

There are two things that would completely smooth things out for my workflows.
1. Auto-saving: There's a difference in the idea of a scratchpad, versus a notes application. In my mind, scratchpads are ephemeral. You write what you want and it's on your scratchpad until you delete it. Explicitly saving shouldn't even be necessary, otherwise it becomes more of a notes application. – At the very least, it'd be okay if CMD+S would save, instead of CMD+Enter.
2. Window focus: Often when writing in a scratchpad, I need to reference other things. Currently if I focus another window, the scratchpad closes, and it's 3 actions to re-open it (open Alfred, re-select the "pad" keyword, re-select the pad I was on) – It'd be awesome if the pad would remain open until it's closed with escape, or triggering the Alfred hotkey again.

I'm not super familiar with Alfred's workflow capabilities, so these may not even be possible at all, and if not, totally understandable!

In short, the Scratchpad workflow is awesome, you're awesome, and these are just suggestions that I think would make it more awesome.

**Comment by zeitlings** [comment]
Hey @beauwest 
Thanks for the kind words!

I'm aware of the issues you have mentioned (auto-saving and pinning the text view), but unfortunately those are things beyond my control. These issues exist due to limitations inherent to how Alfred handles things. It would be useful if you reposted your requests in the [Alfred forum](https://www.alfredforum.com/topic/21812-scratchpad/#comment-113328) so that the devs see that there is a demand for these features.

**Comment by beauwest** [comment]
I kind of figured that might be the case. I'll see about re-posting there for visibility. Thank you for the response!
